src/model/wallet.ts

```diff
@@ -339,17 +370,14 @@ export const signTransaction = async ({
       return { result };
     } catch (error) {
       Alert.alert(lang.t('wallet.transaction.alert.failed_transaction'));
-      logger.sentry('Error', error);
-      const fakeError = new Error('Failed to sign transaction');
-      captureException(fakeError);
+      logger.error(new RainbowError('Failed to sign transaction'), { error });
       return { error };
     }
   } catch (error) {
     Alert.alert(lang.t('wallet.transaction.alert.authentication'));
-    logger.sentry(
-      'Failed to SIGN transaction due to authentication, alerted user'
-    );
-    captureException(error);
+    logger.error(new RainbowError('Failed to sign transaction due to auth'), {
+      error,
+    });
```

Comment:
Will any of these caught `error`s have any PII? Just wanna make sure we don't send any accidentally.

Reply:
this all should be

Reply:
talked to bruno about this and we want all of these errors, all error types look chill tho